apple/Sources/Sargon/Extensions/Swiftified/Profile/Factor/FactorSource/FactorSource+Swiftified.swift

```diff
@@ -69,6 +72,9 @@ extension FactorSource: BaseFactorSourceProtocol {
 			case var .trustedContact(source):
 				source.common = newValue
 				self = .trustedContact(value: source)
+			case var .passphrase(source):
```

Comment:
@micbakos-rdx can you add this Swift unit test for the Passphrase FactorSource, should be **trivially** translated: https://github.com/radixdlt/sargon/blob/main/apple/Tests/TestCases/Profile/Factor/ArculusCardFactorSourceTests.swift#L8-L81